apps/address-book/package.json

```diff
@@ -14,7 +15,7 @@
     "ganache-cli:test": "sh ../../node_modules/@aragon/test-helpers/ganache-cli.sh",
     "lint": "solium --dir ./contracts",
     "precommit": "lint-staged",
-    "prepublishOnly": "truffle compile",
+    "prepublishOnly": "truffle compile --all && npm run abi:extract -- --no-compile",
```

Comment:
`--all` seems redundant here, we have been testing without the option in the about app and it just works fine.

Reply:
for consistency we may want to call the compile script, as in the rest of apps, anyway not a big deal 
```suggestion
    "prepublishOnly": "npm run compile && npm run abi:extract -- --no-compile",
```